src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -2129,6 +2172,21 @@ public GHHook createWebHook(URL url) throws IOException {
         return createWebHook(url, null);
     }
 
+    /**
+     * Update web hook gh hook.
+     *
+     * @param id
+     *            the ID of the hook to be updated
+     * @param url
+     *            the url
+     * @return the gh hook
+     * @throws IOException
+     *             the io exception
+     */
+    public GHHook updateWebHook(long id, URL url) throws IOException {
```

Comment:
:warning: All overloaded methods should be placed next to each other. Placing non-overloaded methods in between overloaded methods with the same type is a violation. Previous overloaded method located at line '2,158'. (#QR505:2186#)-CHECKSTYLE_OverloadMethodsDeclarationOrderCheck